cmake/ConkyBuildOptions.cmake

```diff
@@ -26,15 +26,15 @@ if(NOT CMAKE_BUILD_TYPE)
     set(
       CMAKE_BUILD_TYPE Debug
       CACHE
-      STRING
-      "Choose the type of build, options are: None Debug Release RelWithDebInfo MinSizeRel."
+        STRING
```

Comment:
It looks like the only thing that changed in this file is the formatting. I think it would be better to preserve the formatting here.